src/algebra/group_ring_action.lean

```diff
@@ -65,6 +65,14 @@ def mul_semiring_action.to_ring_equiv [mul_semiring_action G R] (x : G) : R ≃+
 { .. distrib_mul_action.to_add_equiv R x,
   .. mul_semiring_action.to_ring_hom G R x }
 
+/-- A multiplicative action of `N` on `R` and a group homomorphism `K →* N` induce
+a multiplicative action of `K` on `R`. -/
+def mul_semiring_action.comp_hom (g : K →* N) [mul_semiring_action N R] :
```

Comment:
Let's try to make the docstring match the almost-identical `mul_action.comp_hom`:
```suggestion
/-- Compose a `mul_semiring_action` with a `monoid_hom`, with action `f r' • m`.
See note [reducible non-instances]. -/
@[reducible] def mul_semiring_action.comp_hom (g : K →* N) [mul_semiring_action N R] :
```

Also note this ought to be `reducible` in order to match` mul_action.comp_hom`.